# General settings:

arm_velocity_scaling_search: 0.5
arm_velocity_scaling_grasp: 0.2
pregrasp_offset_z: -0.05

# Waypoints for the base to follow during search action:

search_waypoints:
  initial:
    - [15.6757316589, -0.985509037971, -1.14762610015]
    - [18.4454135895, -1.62419974804, -0.293683380035]
    - [21.1538028717, -1.79279625416, 6.57794558821]

  loop:
    - [22.884, -0.478, 63.0]
    - [23.3571300507, 0.906202673912, 91.400792341]
    - [23.2464008331, 2.67486047745, 89.8622512493]
    - [23.2799835205, 4.79860496521, 107.616658638]

drop_waypoints:
  retract_waypoint: [22.6979408264, -0.706754267216, -136.143811141]
  drop_waypoints: 
    - [18.8157615662, -1.54435634613, 174.374674869]
    - [16.5263442993, -1.03566563129, -179.855710433]
    - [10.8428039551, -1.36035263538, 178.827247868]
    - [4.33228302002, -1.1852132082, 178.782756016]
